continuous-integration/linux/lint.sh

```diff
@@ -41,6 +41,7 @@ cmake -GNinja \
     -DCMAKE_CXX_CLANG_TIDY="/usr/bin/clang-tidy-10" \
     -DWARNINGS=ON \
     -DWARNINGS_AS_ERRORS=ON \
+    -DUSE_ARUCO=ON \
```

Comment:
Should we set all of them on for linting?

Reply:
I can do it. What do you think? I think it makes sense. Then, in future, if we disable some by default, the liner will do its job anyway for all the files.

Reply:
Yeah, good point! I should have thought of that since I was the one suggesting it for `build.sh` :grinning:

Reply:
done

Reply:
https://github.com/zivid/zivid-cpp-samples/pull/169/commits/61123d530ec7c7034e74b4893cbe5f2178dd22cf

I added "lint" to the commit message.